ts/features/bonus/cgn/screens/activation/CgnCTAStartActivationScreen.tsx

```diff
@@ -32,7 +36,7 @@ const BpdCTAStartOnboardingScreen: React.FC<Props> = (props: Props) => {
   React.useEffect(() => {
     // cgnActivationStart navigate to ToS screen that needs cgb bonus from available bonus list
     if (availableBonus.length > 0 && cgnBonus) {
-      navigateBack();
+      // navigateBack();
```

Comment:
Can this commented code be removed?

Reply:
this has been removed as a part updating the branch with code on master branch